### PR TITLE
Create desktop version for the mobile portfolie-pull request#4

### PR DESCRIPTION
**Pull request by efmomoh**
This is the desktop version of my portfolio project. 
it was built upon HTML-CSS-media queries.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -107,15 +102,131 @@ <h3>Enssah</h3>
...
+              <div class="btn-lang" id="btn-lang-2">
+                <ul>
+                  <li><button class="lang-btn" type="button">HTML</button></li>
+                  <li><button class="lang-btn" type="button">CSS</button></li>
+                  <li>
+                    <button class="lang-btn" type="button">JavaScript</button>
+                  </li>
+                </ul>
+              </div>
+              <button class="button" type="button">See Project</button>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly fix the border radius given to the button

File: index.html [hunk 2]
```diff
@@ -160,193 +275,97 @@ <h3>Enssah</h3>
...
-          src="./portfolio-icons/Snapshoot Portfolio-3.jpg"
-          alt="Snapshoot Photo"
-        />
-        <div class="flex">
-          <strong>Momoh</strong>
-          <div class="canopy-section">
+    </section>
+    <!--END OF WORK SECTIONS-->
+    <!-- START OF ABOUT MYSELF PAGE -->
+    <section class="about-section" id="my-page">
```
Comment (Kossi-stack) [inline index.html:277]: - [ ] Kindly align the about me session well by using padding-left.
- [ ] Your Design
Screenshot 2023-04-07 at 16 54 00

Figma  Design
Screenshot 2023-04-07 at 16 56 26

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @efmomoh,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css [hunk 3]
```diff
@@ -710,15 +521,234 @@ button:hover,
...
+  .form-box {
+    padding: 320px;
+    margin-top: -350px;
+  }
+
+  .contact-me {
+    width: 70%;
+    margin-left: 130px;
+  }
+
```
Comment (shella12) [inline style.css:734]: - [ ] [OPTIONAL] Great job on your portfolio it looks great almost identical to the figma, however I noticed some blank space after the contact form in desktop version. I would highly suggest you to fix it as it makes your design look sloppy.

Screenshot 2023-04-08 at 11 08 30 AM

**Review by shella12 — APPROVED** [review]
Hi @efmomoh , 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Great design implementation 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (efmomoh) [reply style.css:734]: @shella12 thanks a lot for the feedback. I will implement it.

File: index.html (hunk 2 above)
Comment (efmomoh) [reply index.html:277]: @Kossi-stack  thanks for the feedback. I will make those adjustments.

**Pull request merged**